practice-two/src/stores/reducers.js

```diff
@@ -0,0 +1,28 @@
+export const initialCustomer = {
+  customers: [],
+};
+
+export const customerReducer = (state, action) => {
+  switch (action.type) {
+    case 'CREATE_CUSTOMER': {
```

Comment:
create action type for constans

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/c2434827-3045-47a5-bbfd-f4ea4638f637)
![image](https://github.com/datdragon777/React-Training/assets/74806826/7fc22c7c-3c0b-4bbc-8fa1-dc255b4bc411)
![image](https://github.com/datdragon777/React-Training/assets/74806826/417ec676-8e21-426f-b64e-99f29a133ae3)